src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -694,5 +704,159 @@ void addSiteWithDuplicateDomainNames(Vertx vertx, VertxTestContext testContext)
         });
     }
 
+    @Test
+    void appNameNoSiteId(Vertx vertx, VertxTestContext testContext) {
+        fakeAuth(Role.MAINTAINER);
```

Comment:
Would be good to have a test that shows roles other than MAINTAINER and SHARING_PORTAL can't call these APIs.

Reply:
I added a new test `appNameRoleUnauthorized` with role `MAPPER`